packages/docusaurus-init/templates/classic-typescript/tsconfig.json

```diff
@@ -4,7 +4,8 @@
   "compilerOptions": {
     "paths": {
       "@site/*": ["./*"]
-    }
+    },
+    "baseUrl": "."
   },
   "include": ["src/"]
```

Comment:
are baseUrl + include really necessary? baseUrl is in base tsconfig and include has been removed by orta

also `@site/` alias can be removed now?

Reply:
`baseUrl` is actually required; `include` I don't know. But putting it there makes me feel safe as other files won't be accidentally included

`@site/` is used in the `versions.tsx` page and removing it causes errors

Reply:
hmmm it seems I got it working by keeping just `baseUrl`